content/es/docs/reference/glossary/secret.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Secret
+id: secret
+date: 2019-05-16
+full_link: /docs/concepts/configuration/secret/
+short_description: >
+  Almacena información sensible, como contraseñas, tokens OAuth o claves ssh.
+
+aka: 
+tags:
+- core-object
+- security
+---
+ Almacena información sensible, como contraseñas, tokens OAuth o claves ssh.
+
+<!--more--> 
+
+Ofrece un mayor control sobre cómo usar información sensible y reduce el riesgo de exposición accidental, incluyendo [encriptado](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted) en reposo. Un {{< glossary_tooltip text="Pod" term_id="pod" >}} referencía el secreto cómo un simple fichero en un ??volume mount?? o ??by the kubelet pulling images for a pod?? (desconozco el mecanismo, no sé traducirlo). Los secretos son idóneos para datos confidenciales y los [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/) para datos no confidenciales.
```

Comment:
Tengo un par de dudas:

 - `??volume mount??`
 - `??by the kubelet pulling images for a pod??`